### DISCUSSION TITLE
Download won't start

### DISCUSSION THREAD

**Discussion by Schaarenberg** [TumblThreeApp/TumblThree#435] [Q&A (Question & Answer)]
Hello,

my download does not start no matter how i try it.
I have tried all possible and thinkable ways, deleted AppData/Local/TumblrThree, reinstalled TT, tried different URLs etc. but nothing works.

TT recognises the number of downable content and also a few other "stats" or colums after pasting the url.
But when i click "Download" it wont work, it dosent even make a "move" somewhere. Even after 5 mins.

Im out of ideas what to try where and how now.
Do i miss somthing?

Hope u can help me with this somewhat less helpful information.

best regards :)

**Comment by thomas694** [comment]
Hello,
have you also added the blog to the Queue by context menu, double click or drag & drop?
